qt/applications/workbench/workbench/plotting/plotscriptgenerator/__init__.py

```diff
@@ -68,6 +69,14 @@ def generate_script(fig, exclude_headers=False):
             if not curve_in_ax(ax):
                 continue
             plot_commands.extend(get_plot_cmds(ax, ax_object_var))  # ax.plot
+
+        if not isinstance(ax.xaxis.minor.locator, NullLocator):
+            plot_commands.append("axes.minorticks_on()")
+
+            if hasattr(ax, 'show_minor_gridlines'):
+                plot_commands.append(f"axes.show_minor_gridlines = {ax.show_minor_gridlines}")
+
```

Comment:
These lines should be in the specific files in this package (e.g. axes.py) not really in the init.py.  They all look associated with the axes so they shouild be in axes.py. Otherwise this code will get too messy over time.